client/me/purchases/purchase-item/index.jsx

```diff
@@ -55,65 +49,95 @@ class PurchaseItem extends Component {
 		);
 	}
 
-	renewsOrExpiresOn() {
+	getStatus() {
 		const { purchase, translate, moment } = this.props;
+		const expiry = moment( purchase.expiryDate );
 
-		if ( showCreditCardExpiringWarning( purchase ) ) {
-			return (
-				<Notice isCompact status="is-error" icon="notice">
-					{ translate( 'Credit card expiring soon' ) }
-					{ this.trackImpression( 'credit-card-expiring' ) }
-				</Notice>
-			);
+		if ( purchase && isPartnerPurchase( purchase ) ) {
+			return translate( 'Managed by %(partnerName)s', {
+				args: {
+					partnerName: getPartnerName( purchase ),
+				},
+			} );
 		}
 
 		if ( isRenewing( purchase ) && purchase.renewDate ) {
 			const renewDate = moment( purchase.renewDate );
 
-			return translate( 'Renews at %(amount)s on %(date)s.', {
+			if ( creditCardHasAlreadyExpired( purchase ) ) {
+				return (
+					<span className="purchase-item__is-error">
+						{ translate( 'Credit card expired' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **26** times:
`translate( 'Expired Credit Card' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/purchase-listing-view)